apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/charSequenceContainsCreators.kt

```diff
@@ -190,6 +190,32 @@ fun <T : CharSequence> CharSequenceContains.CheckerOption<T, NoOpSearchBehaviour
     vararg otherPatterns: String
 ): Expect<T> = addAssertion(ExpectImpl.charSequence.contains.regex(this, pattern glue otherPatterns))
 
+/**
+ * Finishes the specification of the sophisticated `contains` assertion where the given regular expression [pattern]
+ * as well as the [otherPatterns] are expected to have a match, using a non disjoint search.
+ *
+ * By non disjoint is meant that `'aa'` in `'aaaa'` is found three times and not only two times.
+ * Also notice, that it does not search for unique matches. Meaning, if the input of the search is `'ab'` and [pattern]
+ * is defined as `'a(b)?'` and one of the [otherPatterns] is defined as `'a(b)?'` as well, then both match, even though
+ * they match the same sequence in the input of the search. Use an option such as [atLeast], [atMost] and [exactly] to
+ * control the number of occurrences you expect.
+ *
+ * Meaning you might want to use:
+ *   `contains.exactly(2).regex('a(b)?')`
```

Comment:
adjust the example, use Regex

Reply:
I had thought of changing `contains.exactly(2).regex('a(b)?')` to `contains.exactly(2).regex(Regex("""a(b)?"""))`, but the KDoc had used `'` instead of `"` for denoting `a(b)?`, so I thought it was pseudocode. If I understood correctly, I should also fix the quotes for the original `String` method's KDoc too, right?

Reply:
good catch, yes please also change the string example. please don't use triple quotes for Regex though. single quotes fit better here.